skt/kerneltree.py

```diff
@@ -274,6 +285,17 @@ def __get_remote_name(self, uri):
         return remote_name
 
     def merge_git_ref(self, uri, ref="master"):
+        """
+        Merge a git branch/reference from source kernel Git reference.
```

Comment:
The source mentioned here is still a bit confusing. It's great that the argument contains the description (and please keep it there!) but I'd simplify this sentence to just say "Merge a git branch/reference into the tree.", to make it less confusing. What do you say?